src/settings/components/Sets/SetsForm/components/GeneralInformation/components/SetSpecification/SetSpecification.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  setSpecValueFromFilteringConditions,
+} from '../../../../../../../util';
+
+import {
+  SET_FIELDS,
+  SET_FIELDS_INITIAL_VALUES,
+} from '../../../../../../../constants';
+
+import css from './SetSpecification.css';
+
+const generateSetSpecification = (value) => (
+  value && value.length
```

Comment:
I suggest using isEmpty from lodash.
```suggestion
  isEmpty(value)
```

or `?.`

```suggestion
  value?.length
```

Reply:
Fixed